pkgs/development/tools/node-webkit/nw12.nix

```diff
@@ -9,9 +9,9 @@ let
   nwEnv = buildEnv {
     name = "nwjs-env";
     paths = [
-      xorg.libX11 xorg.libXrender glib gtk atk pango cairo gdk_pixbuf
-      freetype fontconfig xorg.libXcomposite alsaLib xorg.libXdamage
-      xorg.libXext xorg.libXfixes nss nspr gconf expat dbus stdenv.cc
+      xorg.libX11 xorg.libXrender glib gtk atk pango.out cairo gdk_pixbuf
+      freetype fontconfig.lib xorg.libXcomposite alsaLib xorg.libXdamage
+      xorg.libXext xorg.libXfixes nss nspr gconf expat dbus.lib
       xorg.libXtst xorg.libXi xorg.libXcursor xorg.libXrandr libcap
       libnotify
     ];
```

Comment:
Alternatively, set `extraOutputsToInstall = [ "lib" "out" ]` and leave the other stuff as it was, perhaps changing `stdenv.cc` to `stdenv.cc.cc`.